nixarr/sabnzbd/default.nix

```diff
@@ -0,0 +1,245 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.nixarr.sabnzbd;
+  defaultPort = 8080;
```

Comment:
Can this be added as an option instead? The reason the "Arrs" don't have it is because their configuration format is really dumb. But that doesn't seem to be the case here?

Reply:
Resolved with bc85cdc